### PR TITLE
search bar component added

### PR DESCRIPTION
**Pull request by Aaronkitiibwa**
Search bar component added

This will help our guests to search for the our available services.

The search results shall be generated based on key words that are related to our food categories that we provide.

### CODE REVIEW

File: NutriLife/src/components/SearchBar.jsx [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+
+const[searchInput, setSearchInput] = useState("null")
+
+const handleChange = (e) => {
+    e.preventDefault();
+    setSearchInput(e.target.value);
+};
+
+return(
+    <MantineProvider>
```
Comment (Yesigomwe) [inline NutriLife/src/components/SearchBar.jsx]: The MantineProvider tag is supposed to be the root tag for Home.jsx file and not other components. Other components should have just either a parent div tag or fragment.

File: NutriLife/src/components/SearchBar.jsx [hunk 2]
```diff
@@ -0,0 +1,26 @@
+import { useState } from "react"
+import { MantineProvider } from "@mantine/core";
+const searchBar = () =>{
```
Comment (Yesigomwe) [inline NutriLife/src/components/SearchBar.jsx]: All component names should start with a capital letter. So this should be SearchBar

**Review by Yesigomwe — COMMENTED** [review]
Thank you Aaron for the PR. I have left some comments in your code after I ran the branch and I could not see your work. Please check them. Are you able to run the code you wrote and see the search bar?

**Comment by Aaronkitiibwa** [comment]
Feedback received. Let me work on it

**Comment by Yesigomwe** [comment]
@Aaronkitiibwa  I have feedback for you in the Searchbar.jsx file. Look at the comment

**Pull request closed without merging**